guides/release/typescript/core-concepts/ember-data.md

```diff
@@ -8,10 +8,7 @@ We do _not_ cover general usage of EmberData; instead, we assume that as backgro
       <div class="cta-note-heading">Zoey says...</div>
       <div class="cta-note-message">
         <p>
-          These guides currently assume you are using the EmberData <code>@types</code> packages in conjunction with the Ember <code>@types</code> packages.
-        </p>
-        <p>
-          For improved (albeit less stable) types, you can switch to <a href="https://github.com/emberjs/data/blob/main/guides/typescript/index.md">EmberData's alpha native types, documented at this link</a>. Using the EmberData alpha native types will also require <a href="https://blog.emberjs.com/stable-typescript-types-in-ember-5-1/">switching to the Ember native types</a>, which are guaranteed to always be 100% correct and 100% up to date!
+        The following content applies to the native EmberData types, which are currently considered "unstable" (though in practice, they've been pretty stable as of late). These guides may change as the EmberData types are finalized.
```

Comment:
NOTE: I only updated the existing docs to make sure they weren't wrong. I did not add any new docs re: requests, builders, includes, etc.